sdk/communication/CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+
+Depending on which package you are testing, it may need special environment variables to test succesfully. In each package, there is a `recordedClient.ts` file; In that file you will find `const replaceableVariables` with the names of the variables the package uses. Make sure to set these variables before running the tests themselves.
+
+After setting up the environment variables and setting `TEST_MODE` to `LIVE`, you can run the tests in the Test Explorer.
+
+### Record mode
+
+RECORD mode is similar to LIVE mode because it also hits an actual resource. In addition to hitting the resource, RECORD mode will also record the successful calls to the service in json format. As mentioned, the recordings are stored in the test package under `recordings\browsers` and `recordings\node`.
+
+If you would like to generate new recordings for a test, setting `TEST_MODE` to `RECORD`.
+
+### Managed Identity Tests
+
+If you ran the tests in RECORD mode, you may have noticed that the files inside the `recordings` folder were updated. If any of the tests failed, you will see the error message in your terminal logs.
+
+The most probable thing is that the managed identity tests will fail at first. This is because we haven't set up any managed identity credentials for the `DefaultAzureCredential` object inside the tests to reference to. Managed identity tests do not execute in `browser` mode because of a security limitation using `DefaultAzureCredential`.
+
+There are multiple ways of creating a managed identity credential. One of the easiest ways is to install the [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli) and run the `az login` command. If you are listed as a contributor of the resource you are testing against, this should be enough for the DefaultAzureCredential object to get the corresponding Azure Active Directory credentials you need.
+
+Another way to authenticate is to set up 3 environment variables called `AZURE_CLIENT_ID`, `AZURE_TENANT_ID` and `AZURE_CLIENT_SECRET` and set their values to the ones from a registered Azure Active Directory application that is linked to the resource you are testing against.
+
+If you are testing against a personal resource, you can check the [Managed Identity Quickstart Guide for ACS](https://docs.microsoft.com/azure/communication-services/quickstarts/managed-identity-from-cli) for an easy ramp-up process.
+
+For a more in-depth look on how to authenticate using managed identity, refer to the [Azure Identity client library for JavaScript](https://docs.microsoft.com/en-us/javascript/api/overview/azure/identity-readme?view=azure-node-latest) documentation. This document also has more ways for you to authenticate using the `DefaultAzureCredential` object besides the ones we discussed in this contributing file.
```

Comment:
https://docs.microsoft.com/en-us/javascript/api/overview/azure/identity-readme?view=azure-node-latest 

The link here should not have en-us